frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -459,9 +488,27 @@ const ManageHostsPage = ({
     retrieveHostCount(options);
   };
 
+  let teamSync = false;
+  if (currentUser && availableTeams) {
+    // team_id starts as a string and needs to be parsed. If the user provides a non-numeric
+    // string, it will parse as NaN. However, we need to handle if team_id is undefined (which is
+    // would be the case for the root path to all teams) because undefined also parses to NaN.
+    // We want to be able to check that teamIdParam equals undefined in the `else if` below because
+    // that will correspond to the undefined state `currentTeam`, which represents the root path.
```

Comment:
Could this be edited down to a single line comment? The code reads pretty clear to me so I don't think the explanation is necessary.